jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/handler/TomcatAppSeverHandlerTest.java

```diff
@@ -15,65 +15,69 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.Collections;
 
 import org.eclipse.jkube.generator.api.GeneratorContext;
 import org.eclipse.jkube.kit.common.Plugin;
 
 import mockit.Expectations;
 import mockit.Mocked;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertTrue;
 
-public class TomcatAppSeverHandlerTest {
+@SuppressWarnings({"unused", "ResultOfMethodCallIgnored"})
+class TomcatAppSeverHandlerTest {
 
-  @Rule
-  public TemporaryFolder temporaryFolder = new TemporaryFolder();
+  @TempDir
+  Path temporaryFolder;
 
   @Mocked
   private GeneratorContext generatorContext;
 
   @Test
-  public void isApplicableHasContextXmlShouldReturnTrue() throws IOException {
+  void isApplicableHasContextXmlShouldReturnTrue() throws IOException {
     // Given
     // @formatter:off
     new Expectations() {{
-      generatorContext.getProject().getBuildDirectory(); result = temporaryFolder.getRoot();
+      generatorContext.getProject().getBuildDirectory(); result = temporaryFolder.toFile();
     }};
     // @formatter:on
-    assertTrue(new File(temporaryFolder.newFolder("META-INF"), "context.xml").createNewFile());
-    assertTrue(new File(temporaryFolder.newFolder("META-INF-1337"), "context.xml").createNewFile());
+    File baseDir = Files.createDirectory(temporaryFolder.resolve("META-INF")).toFile();
+    assertThat(new File(baseDir, "context.xml").createNewFile()).isTrue();
```

Comment:
As commented [here](https://github.com/eclipse/jkube/issues/1563#issuecomment-1203719846), these assertions are not necessary if you use the nio Files utility.